lib/atat-net-stack.ts

```diff
@@ -55,12 +62,17 @@ export class AtatNetStack extends cdk.Stack {
         })
       ),
     });
-    this.vpc = vpc;
-    this.outputs.push(
-      new cdk.CfnOutput(this, "VpcId", {
-        value: this.vpc.vpcId,
-      })
-    );
+    const dnsLogsGroup = new logs.LogGroup(this, "VpcDnsQueryLogs", {
+      retention: logs.RetentionDays.INFINITE,
+    });
+    // Capture all DNS queries made by all hosts in the VPC
+    const dnsLoggingConfig = new route53Resolver.CfnResolverQueryLoggingConfig(this, "DnsLogging", {
+      destinationArn: dnsLogsGroup.logGroupArn,
+    });
+    const vpcDnsLogging = new route53Resolver.CfnResolverQueryLoggingConfigAssociation(this, "VpcDnsLogging", {
+      resolverQueryLogConfigId: dnsLoggingConfig.attrId,
+      resourceId: vpc.vpcId,
```

Comment:
Cool! So this is lower level logging? Are the logs from the resolver similar (or the same) logs that you were showing us in `Logs Insights`?